user-data/user_data.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
`user_data.sh` is project-specific.
should be configurable

Reply:
It is configurable via the "user_data" variable.